ansible-advancedlinux/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
   "itemDisplayName": "Deploy N Linux VMs and set up Ansible so you can easily manage all the VMs",
-  "description": "Manage your  Linux VMs efficiently using Ansible.This advanced template deploys N number of Linux VMs and it configures Ansible and SSH",
-  "summary": "This advanced template deploys N number of Linux VMs and it configures Ansible so you can easily manage all the VMS. Don't suffer more pain configuring and managing all your VMs , just use Ansible! Ansible is a very powerful masterless configuration management system based on SSH.Do you have 1 or 10 or 100 VMs? It won't make any difference with Ansible",
+  "description": "Manage your  Linux VMs efficiently using Ansible.This advanced template deploys N number of Linux VMs and it configures Ansible and SSH.Then using an Ansible playbook it will create a RAID with all the available data disks",
+  "summary": "This advanced template deploys N number of Linux VMs and it configures Ansible so you can easily manage all the VMS.Additionally,it will execute an ansible Playbook to create a RAID with all the available data disks. Don't suffer more pain configuring and managing all your VMs , just use Ansible! Ansible is a very powerful masterless configuration management system based on SSH.Do you need 1 or 10 or 100 VMs? It won't make any difference with Ansible",
```

Comment:
@GONZALORUIZ Can you reduce this to less than 200 characters? You can use description for the long text up to 1000 characters.